sql/wsrep_mysqld.cc

```diff
@@ -1857,8 +1857,31 @@ int wsrep_to_buf_helper(THD *thd, const char *query, uint query_len,
 
   /* continue to append the actual query */
   Query_log_event ev(thd, query, query_len, false, false, false, 0);
+
+  /* Below commands will enter the row in mysql.password_history table.
```

Comment:
Probably `insert` is more appropriate here?
```suggestion
  /* Below commands will insert the row in mysql.password_history table.
```